db/models/invoice.go

```diff
@@ -14,6 +14,7 @@ type Invoice struct {
 	UserID               int64        `json:"user_id" validate:"required"`
 	User                 *User        `bun:"rel:belongs-to,join:user_id=id"`
 	Amount               int64        `json:"amount" validate:"gte=0"`
+	Fee                  int64        `json:"fee" validate:"gte=0"`
```

Comment:
can this be zero here? I think we save the invoice before we know the fee amount. ?

Reply:
It can be zero if paid to direct channel partner or internal.
I added it so it could be passed on to the `HandleSuccessfulPayment` func.

Reply:
sorry I meant `NULL`

Reply:
Oh, like that. Yes I think we could make it nullable to differentiate between a not-yet-known fee and a 0 fee.